test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -184,6 +185,33 @@ public void GetSchemaFromDatabaseQualifiedNameWithInvalidNameTest()
             string actualSchemaName = peekDefinition.GetSchemaFromDatabaseQualifiedName(validDatabaseQualifiedName, objectName);
             Assert.Equal(actualSchemaName, expectedSchemaName);
         }
+
+        /// <summary>
+        /// Test Deletion of peek definition scripts for a valid temp folder that exists
+        /// </summary>
+        [Fact]
```

Comment:
I'd really like to see more unit tests that check the various branches of code that were added here. Especially the branches for using TempPath and what happens on service shutdown.